scripts/setup_gae.py

```diff
@@ -41,13 +41,9 @@ def main(args=None):
     coverage_home = os.path.join(
         common.OPPIA_TOOLS_DIR, 'coverage-%s' % common.COVERAGE_VERSION)
 
-    # Note that if the following line is changed so that it uses webob_1_1_1,
-    # PUT requests from the frontend fail.
```

Comment:
Are existing e2e tests sufficient to verify this issue no longer exists with the newer webapp/webob dependencies?

Reply:
Yep! The webob error mainly occurs for any of the put requests, post requests, or get requests. In my 'update google app engine' PR, I actually tried removing webob and scripts.start did not even successfully startup the dev server. These issues no longer exist using the new version of webapp2.

Furthermore the frontend tests pass when using put, as can be shown in exploration-data.service.spec.ts.

Reply:
Awesome, thanks for the additional context!